data_steward/tools/add_hpo.py

```diff
@@ -221,6 +287,11 @@ def main(hpo_id, org_id, hpo_name, bucket_name, display_order, addition_type):
                         f'Proceeding to add site.')
             add_lookups(hpo_id, hpo_name, org_id, bucket_name, display_order)
 
+            LOGGER.info(
+                f'hpo_site_id_mappings table successfully updated. Updating `pipeline_table.site_masking` '
+                f'table')
+            update_site_masking_table()
+
 
 if __name__ == '__main__':
     import argparse
```

Comment:
If you add `pipeline_logging` you can add `pipeline_logging.configure(level=logging.DEBUG, add_console_handler=True)` here